### ISSUE TITLE
Collapsible Container / Element result in Javascript Error

### ISSUE THREAD

**Issue by max951** [t3solution/t3sbootstrap#294]
I added a Collapsible Container (uid139), a Collapsible Element (uid140) and a TextMedia Elemt (uid141)
This resulted in an Javsacript error
___
Uncaught TypeError: Cannot read properties of undefined (reading 'parent')
at mt._initializeChildren (bootstrap.min.js?1668622072:6:20078)
    at new mt (bootstrap.min.js?1668622072:6:17969)
    at t3sbCollapsible (t3sbootstrap_04caccca39.js?1671373204:132:23)
    at HTMLDocument.<anonymous> (t3sbootstrap_04caccca39.js?1671373204:346:3)
_initializeChildren @ bootstrap.min.js?1668622072:6
mt @ bootstrap.min.js?1668622072:6
t3sbCollapsible @ t3sbootstrap_04caccca39.js?1671373204:132
(anonym) @ t3sbootstrap_04caccca39.js?1671373204:346
___

I'm not an expert, but I think the error is in line 17-19 of file Resources\Private\Templates\Container\CollapsibleContainer.html
Special in part `document.getElementById('collapse-'+idArr[1])`.
In my case this should get an object with id 'collpase-139'. But this doesn't exist.
c139 has as id 'accordion-139'

so i changend the line 16 of the above file from
`if (accordionButton !== null && accordionButton !== '') {`
to
`if (accordionButton !== null && accordionButton !== '' && document.getElementById('collapse-'+idArr[1]) !== null) {`
and it seems all worked fine again.

Used versions:
TYPO3 11.5.19
t3sbootstrap 5.2.8
PHP 7.4.33

**Comment by t3solution** [comment]
Do I understand correctly that you want to open the "Collapsible Element" with ID 139 with an external link?
Then you also have to select the Collapsible Element and not the Collapsible container as link target!

**Comment by max951** [comment]
No. This already happened on the page where the content elements are inserted. An external link is not used.

I just tried to recreate this in a completely new, empty installation. And the error does not occur. So there must be something else involved in the other project.

I will check again in the project, in which the error occurs and report back here.

**Comment by t3solution** [comment]
This can only be the case if you are using a location.hash in your URL.

**Comment by max951** [comment]
Yes, you're right.

I used the "View"-Button in the edit menubar in the backend to see my changes quickly. 
This brings up (in the blank project) the URL https://p373268.webspaceconfig.de/typo3/t3sb-collapsible#c6 in the browser and ends up with the javasript error I prescribed above.

In the development project the URL https://zwr.hostingmax.de/unser-wasser/wasserqualitaet-und-analyse#c139 will be called and result also in this error. All further javascript will be stopped and in this case the footer will stay layered over the content. 

If I change the source like prescribed above, all things will be fine. Even a external link to #c140 (the Collapsible Element) works well.
